pkgs/by-name/ra/rainbowcrack/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, alglib
+, unzip
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8";
+
+  src = fetchurl {
+    url = "http://project-rainbowcrack.com/rainbowcrack-1.8-linux64.zip";
+    hash = "sha256-xMC9teHiDvBY/VHV63TsNQjdcuLqHGeXUyjHvRTO9HQ=";
+  };
+
+  buildInputs = [ alglib unzip ];
+
+  unpackPhase = ''
+    mkdir -p $out/{bin,share/rainbowcrack}
+    ${unzip}/bin/unzip $src -d $out || true
```

```suggestion
    unzip $src -d $out || true
```
Otherwise you may encounter some cross compilation problems